deploy/helm/charts/README.md

```diff
@@ -65,7 +65,7 @@ The following table lists the configurable parameters of the OpenEBS NDM chart a
 | `ndmExporter.clusterExporter.metricsPort`   | The TCP port number used for exposing NDM cluster exporter metrics  | `9100`          |
 | `featureGates.APIService.enabled`       | Enable the gRPC API service of NDM            | `false`                                   |
 | `featureGates.UseOSDisk.enabled`        | Enable feature-gate to use free space on OS disk | `false`                                   |
-| `featureGates.MountChangeDetection.enabled` | Enable feature-gate to detect mountpoint/filesystem changes | `false`                                   |
+| `featureGates.ChangeDetection.enabled` | Enable feature-gate to detect mountpoint/filesystem changes | `true`                                   |
```

Comment:
nit: this flag also enables size change detection
```suggestion
| `featureGates.ChangeDetection.enabled` | Enable feature-gate to detect mountpoint/filesystem/size changes | `true`                                   |
```

Reply:
I have accepted this suggestion into the PR.